src/client/cli/cmd/transfer.cpp

```diff
@@ -163,9 +165,16 @@ std::vector<std::pair<std::string, fs::path>> cmd::Transfer::args_to_instance_an
     instance_entry_args.reserve(args.size());
     for (const auto& entry : args)
     {
+        // this is needed so that Windows absolute paths are not split at the colon following the drive letter
+        if (QRegularExpression{R"(^[A-Za-z]:[\\/].*)"}.match(entry).hasMatch())
```

Comment:
Hey @andrei-toterman, after testing, this is a little problematic with single letter instance names:

```
$ multipass launch -n C
Launched: C
$ multipass transfer -rp C:\Users\ricab\bin\yasm.exe C:/home/ubuntu/yasm
An instance name is needed for either source or destination
```

A proper fix would probably require expanding the command line to deal with ambiguous cases somehow, but I think this could be *mostly* worked around by dropping the forward slash from the regex. People will need the backward slash to specify windows paths, but I think that's OK. It will still not work if the file name in the instance starts with a backward slash, but that's a corner case we can probably live with. WDYT?

Reply:
Yeah, unfortunately instances with single letter names are not differentiated properly by the regex, and I assumed that would be a corner case as well. But your solution makes more sense. Thanks, I'll do it!